composer.json

```diff
@@ -33,11 +33,14 @@
         "symfony/property-info": "^4.3",
         "doctrine/annotations": "^1.8",
         "ramsey/uuid": "^3.8",
-        "webgriffe/amp-elasticsearch": "^2.0",
+        "webgriffe/amp-elasticsearch": "dev-php-8.3-support#6b26a581fc99c6aba5b06fc2eabfe2dec98578c9 as 2.1",
         "pagerfanta/pagerfanta": "^2.4",
         "symfony/deprecation-contracts": "^2.1",
         "amphp/http-server-form-parser": "^1.1"
     },
+    "suggest": {
+        "ext-mbstring": "*"
+    },
```

Comment:
We should move this to the requirements so we can remove the mb_convert_encoding symbol from the composer require checker whitelist.